circle.yml

```diff
@@ -18,4 +19,5 @@ test:
       - xcodebuild -workspace "$XCODE_WORKSPACE" -scheme "$XCODE_SCHEME" ONLY_ACTIVE_ARCH=NO CODE_SIGN_IDENTITY="" CODE_SIGNING_REQUIRED=NO test | tee $CIRCLE_ARTIFACTS/xcode_raw.log | xcpretty --color --report junit --output $CIRCLE_TEST_REPORTS/xcode/results.xml
       - swift build
   post:
-      - bash <(curl -s https://codecov.io/bash)
+    - bundle exec slather coverage --scheme "$XCODE_SCHEME" --workspace "$XCODE_WORKSPACE" --output-directory $CIRCLE_ARTIFACTS --binary-basename SourceryRuntime --binary-basename Sourcery --cobertura-xml Sourcery.xcodeproj
+    - bash <(curl -s https://codecov.io/bash) -f $CIRCLE_ARTIFACTS/cobertura.xml -X voeragepy -X gcov -X xcode
```

Comment:
@Liquidsoul could you please explain flag `voeragepy`?

Reply:
I admit that I just copied it from the [codecov section](https://github.com/SlatherOrg/slather/blob/master/README.md#usage-with-codecov) in the slather readme without looking at what it does.

Reply:
And it seems it should be `coveragepy`😅

Reply:
I've squashed the fix in the commit 2704571

Reply:
Nice catch @Antondomashnev BTW 👍

Reply:
Hah, cool 😄

Reply:
Honestly I'm trying to get the idea why we need additional flags at all here, I've tried to read [documentation](https://circleci.com/docs/1.0/code-coverage/#codecov) and it seems we don't need any parameters by default. is it because of Slather?

Reply:
Yup, this is to make this work as we expect.
With `slather`, we just want codecov to read the cobertura file and use only its content. That's why the `-X` options are there. It's to prevent `codecov` to try to gather coverage information on its own which he fails to do correctly as we can see in the[ current `master`](https://codecov.io/gh/krzysztofzablocki/Sourcery).

Reply:
If you want to see the available options in the script, look to its source [here](https://codecov.io/bash) (if you're not afraid of `bash` 😉)

Reply:
Aha, thanks, now it's all clear 😄